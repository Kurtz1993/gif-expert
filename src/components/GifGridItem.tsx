import { FunctionComponent } from 'react';
import { GifInfo } from '../models/gif-info.model';

const GifGridItem: FunctionComponent<GifInfo> = ({ title, url }) => {
  return (
    <div className="card animate__animated animate__fadeIn">
      <img className="card__img" src={url} alt={title} />
      <p className="card__title">{title}</p>
    </div>
  );
};

export default GifGridItem;
