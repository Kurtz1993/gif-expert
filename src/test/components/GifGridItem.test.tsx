import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('GifGridItem', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<GifGridItem id="" url="" title="" />);

    expect(wrapper).toMatchSnapshot();
  });
});
