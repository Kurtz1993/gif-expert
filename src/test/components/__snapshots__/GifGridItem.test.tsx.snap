// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifGridItem should render correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt=""
    className="card__img"
    src=""
  />
  <p
    className="card__title"
  />
</div>
`;
